src/main/java/core/basesyntax/FruitShopApp.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.reads.CsvFileReader;
+import core.basesyntax.dao.reads.CsvFileReaderImpl;
+import core.basesyntax.dao.writes.CsvFileWriterOrder;
+import core.basesyntax.dao.writes.CsvFileWriterOrderImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.services.FruitMap;
+import core.basesyntax.services.ReportService;
+import core.basesyntax.transactions.OperationHandler;
+import core.basesyntax.transactions.OperationStrategy;
+import core.basesyntax.transactions.impl.BalanceOperationHandler;
+import core.basesyntax.transactions.impl.PurchaseOperationHandler;
+import core.basesyntax.transactions.impl.ReturnOperationHandler;
+import core.basesyntax.transactions.impl.SupplyOperationHandler;
+import java.util.List;
+import java.util.Map;
+
+public class FruitShopApp {
+    public static void main(String[] args) {
+        CsvFileReader readFile = new CsvFileReaderImpl();
+        List<String> readAllLines = readFile.read("input");

```

Comment:
"input", "output" should have a more real meaning. And must be replaced by constants.